scripts/homepage.js

```diff
@@ -40,16 +40,15 @@ $(function () {
         console.log('Pay ' + current_amount);
         var payment_amount = $('#' + current_amount).val() * 100;
         console.log('Starting payment for ' + payment_amount);
-        if (window.ga) {
-            ga('send',
-               'event',
-               release_title + ' ' + release_version + ' Download (Payment)',
-               'Homepage',
-               payment_amount);
-        }
         if (payment_amount < payment_minimum) {
+            if (window.ga) {
+                ga('send', 'event', release_title + ' ' + release_version + ' Payment (Initiated)', 'Homepage', payment_amount);
+            }
             open_download_overlay();
         } else {
+            if (window.ga) {
+                ga('send', 'event', release_title + ' ' + release_version + ' Download (Free)', 'Homepage', payment_amount);
+            }
             do_stripe_payment(payment_amount);
```

Comment:
This is why we review! Also, users can try to start a payment for less than the minimum, so I'd prefer to keep it to see if that happens regularly.